src/PSR7/Exception/Validation/InvalidBody.php

```diff
@@ -17,7 +19,13 @@ public static function becauseBodyDoesNotMatchSchema(
         SchemaMismatch $prev
     ): self {
         $exception          = static::fromAddrAndPrev($addr, $prev);
-        $exception->message = sprintf('Body does not match schema for content-type "%s" for %s', $contentType, $addr);
+        $exception->message = sprintf(
+            'Body does not match schema for content-type "%s" for %s. [%s in %s]',
+            $contentType,
+            $addr,
+            substr($prev->getMessage(), 0, -1),
```

Comment:
Is it needed for some kind of period trimming? I'd rather suggest use rtrim here then, since one cannot guarantee the message format at all

Reply:
I want to remove last '.' in the sentence of the previous error. So rtrim it's a great option. Thank you